docs/dev/basics.rst

```diff
@@ -37,6 +37,10 @@ rerun
 `make update` also applies the patches that can be found in the directories found in
 `patches`; the resulting branch will be called `patched`, while the commit specified in `modules`
 can be referred to by the branch `base`.
+To separately call `make update` was choosen, to let you decide when to update and reset the external
```

Comment:
```suggestion
The separately call `make update` was choosen to let you decide when to update and reset the external
```

Reply:
This sounds like you misread the sentence?